### PR TITLE
Refine DB test

### PR DESCRIPTION
**Pull request by yito88**
https://scalar-labs.atlassian.net/browse/DLT-6134

- Change the test name to `scalardb-test`
- Add README
- Use retry for TransferPreparer
- Use TimeBasedProcessor
- Check if all nodes are alive when CassandraKiller closes

### CODE REVIEW

File: scalardb-test/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Benchmark/Verification Test for Scalar DB
```
Comment (feeblefakie) [inline scalardb-test/README.md]:
```suggestion
# Benchmark and verification for Scalar DB
```

File: scalardb-test/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    - This job requires a client to execute this job and Cassandra cluster or Cosmos DB which can be accessed from the client
+
+2. Build
+    ```console
+    $ gradle shadowJar
+    ```
+
+3. Make your config file based on `benchmark-config.toml` or `verification-config.toml`
+
+4. Kelpie binary (zip) from https://github.com/scalar-labs/kelpie/releases/ and unzip
```
Comment (feeblefakie) [inline scalardb-test/README.md]:
```suggestion
4. Download Kelpie binary (zip) from https://github.com/scalar-labs/kelpie/releases/ and unzip it
```

File: scalardb-test/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - You can also build Kelpie from the source
+
+
+5. Execute a job
+    ```console
+    $ ${KELPIE}/bin/kelpie --config your_config.toml
+    ```
+
+## Workload
+- Transaction
```
Comment (feeblefakie) [inline scalardb-test/README.md]:
```suggestion
- Payment transaction
```

File: scalardb-test/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  - Transfer money between randomly selected 2 accounts
+    1. Start a transaction
+    2. Read 2 accounts
+    3. Calculate new balances
+    4. Update 2 accounts with new balances
```
Comment (feeblefakie) [inline scalardb-test/README.md]:
```suggestion
    4. Update 2 accounts with the new balances
```

File: scalardb-test/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    5. Commit the transaction
+
+- Account
+  - An account has 2 types `0` and `1`
+    - Basically, the type `0` of an account is used
+    - When the source account ID and the destination account ID is the same, the money will be sent from type `0` to `1` of the account ID
+  - The number of accounts can be set at `[test_config]` `num_accounts` in your config file
+
+## Cassandra Killer
+- Cassandra Killer is an injector to make nodes crash randomly and restart the nodes
```
Comment (feeblefakie) [inline scalardb-test/README.md]:
```suggestion
- Cassandra Killer is an injector to kill nodes randomly and restart the nodes
```

File: scalardb-test/README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  - The nodes to be crashed are selected randomly from `[killer_config]` `contact_points`
```
Comment (feeblefakie) [inline scalardb-test/README.md]:
```suggestion
  - The nodes to be killed are selected randomly from `[killer_config]` `contact_points`
```

File: scalardb-test/src/main/java/scalardb/Common.java [hunk 7]
```diff
@@ -1,4 +1,4 @@
-package verification.db.transfer;
+package scalardb;
```
Comment (feeblefakie) [inline scalardb-test/src/main/java/scalardb/Common.java:1]: Not fully sure, but is it better to add `kelpie` or `kelpie-test` in the package ?

**Review by feeblefakie — COMMENTED** [review]
Overall looking good ! Left minor suggestions and a comment.

File: scalardb-test/src/main/java/scalardb/Common.java (hunk 7 above)
Comment (yito88) [reply scalardb-test/src/main/java/scalardb/Common.java:1]: Thank you! Sounds better.
I'll refine the package names for all tests in another PR.

**Review by feeblefakie — APPROVED** [review]
LGTM !

**Pull request merged**